extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -965,6 +976,8 @@ export class AmpA4A extends AMP.BaseElement {
         if (!isArray(metaData.customElementExtensions)) {
           throw new Error('Invalid extensions');
```

Comment:
As long as you're here...  This error message seems vague.  (For all that I know, I wrote it.)  If it's getting thrown here, it will be caught and re-broadcast to StackDriver, right?  Can we say something more informative?